backend/src/main/java/com/bakdata/conquery/util/search/TrieSearch.java

```diff
@@ -193,7 +193,7 @@ private void ensureWriteable() {
 		if (!shrunk) {
 			return;
 		}
-		throw new IllegalStateException("Cannot alter a shrunk search.");
+		throw new IllegalStateException("Cannot alter a shrunk index.");
```

Comment:
:(

Reply:
hatte in intellij einen haken zuviel gesetzt